snuba/settings.py

```diff
@@ -5,23 +5,27 @@ def _load_settings(obj=locals()):
     main Snuba directory, or they can provide a full absolute path such as
     `/foo/bar/my_settings.py`."""
 
+    import importlib
+    import importlib.util
     import os
-    import imp
-
-    path = os.path.dirname(__file__)
 
     settings = os.environ.get('SNUBA_SETTINGS', 'base')
-    if not settings.startswith('/') and not settings.startswith('settings_'):
-        settings = 'settings_%s' % settings
-    if not settings.endswith('.py'):
-        settings += '.py'
 
-    settings = os.path.join(path, settings)
-    settings = imp.load_source('snuba.settings', settings)
+    if settings.starstwith('/'):
```

Comment:
`startswith` (test build is throwing up errors, just saw this)

Reply:
Fixed.